src/actions.rs

```diff
@@ -609,6 +618,28 @@ impl Action {
             .churn_needed
             .map_or(false, |v| v == ChurnNeeded::Split)
     }
+
+    pub fn complete_merge(&self) {
+        self.0.borrow_mut().complete_merge()
+    }
+
+    pub fn has_sibling_merge_info(&self) -> bool {
+        match self.0.borrow().merge_infos {
+            Some(merge_info) => {
+                let our_section = self.our_section().0;
+                let their_section = merge_info.0;
+                (our_section.0 - their_section.0).abs() == 1
+            }
+            None => false,
+        }
+    }
+
+    pub fn merge_sibling_info_to_new_section(&self) -> SectionInfo {
+        let our_section = self.our_section();
+        let their_section = self.0.borrow_mut().merge_infos.take();
+        let their_section = their_section.expect("Merge infos missing").0;
+        SectionInfo(Section((our_section.0).0 + (their_section.0) + 1), 0)
```

Comment:
Same: Add comment to function explaining new convention.
Why + 1?

Reply:
Should it not be /2 to be like normal merge bits.

Reply:
See other comment about section names. Since currently the name is just a number not a prefix, I just picked an arbitrary convention to produce a new name from the two previous.

We can of course switch everything over to be prefixes...I think that should in that case be a follow up PR though